lightly/api/api_workflow_client.py

```diff
@@ -97,7 +97,7 @@ def __init__(
 
         configuration = get_api_client_configuration(token=token)
         self.api_client = ApiClient(configuration=configuration)
-        self.user_agent = f"Lightly/{__version__}/python ({platform.platform()})"
+        self.api_client.user_agent = f"Lightly/{__version__}/python/{platform.python_version()} (platform; {platform.platform()})"
```

Comment:
Following the syntax of https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#syntax

it should rather be;
```
f"Lightly/{__version__} ({platform.system()}/{platform.release()}; {platform.platform()}; {platform.processor()};) python/{platform.python_version()}"
```

wdyt?

Reply:
Makes sense to me, it yields
`Lightly/1.2.43 (Darwin/22.2.0; macOS-10.16-x86_64-i386-64bit; i386;) python/3.9.7`

Reply:
nice